pkg/lib/indexconfig/indexconfig_test.go

```diff
@@ -0,0 +1,108 @@
+package indexconfig
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	manifestDir      = "./testdata/bundle/manifests"
+	tmpTestDirPrefix = "tmp_test_"
+	expectedConfig   = "{\n    \"schema\": \"olm.package\",\n    \"name\": \"lib-bucket-provisioner\",\n    \"defaultChannel\": \"alpha\",\n    \"icon\": {\n       \"base64data\": \"\",\n       \"mediatype\": \"\"\n    },\n    \"channels\": [\n       \"alpha\",\n       \"beta\"\n    ],\n    \"description\": \"\"\n }\n{\n    \"schema\": \"olm.bundle\",\n    \"name\": \"lib-bucket-provisioner.v1.0.0\",\n    \"package\": \"lib-bucket-provisioner\",\n    \"image\": \"\",\n    \"version\": \"1.0.0\",\n    \"properties\": [\n       {\n          \"type\": \"olm.channel\",\n          \"value\": {\n             \"name\": \"alpha\",\n             \"replaces\": \"\"\n          }\n       },\n       {\n          \"type\": \"olm.channel\",\n          \"value\": {\n             \"name\": \"beta\",\n             \"replaces\": \"\"\n          }\n       },\n       {\n          \"type\": \"skips\",\n          \"value\": \"\"\n       },\n       {\n          \"type\": \"skipsRange\",\n          \"value\": \"\"\n       }\n    ],\n    \"relatedImages\": []\n }\n"
```

Comment:
Can we put the expected config fixture (and I'm sure there will be many) in a separate `testdata/bundle/golden` directory as a json-file instead? The plus is since they are human-readable it's easier to understand the test as well.